src/main/java/org/jitsi/jicofo/JitsiMeetConfig.java

```diff
@@ -124,6 +130,12 @@
      */
     public static final int START_BITRATE_DEFAULT = 800;
 
+    /**
+     * The default value for the "minParticipants" property.
+     */
+    private static final int MIN_PARTICIPANTS_DEFAULT = 2;
```

Comment:
It is starting to get messy with more constants being added to this file. Some of them have PNAME, or DEFAULT suffix, some of them don't. Do you mind cleaning this up before things get out of hands ? What do you think about renaming all properties to PNAME_{property name} and DEFAULT_{property name} ? Then if we sort things alphabetically properties will be grouped with properties and defaults with defaults. If that's too much work or you don't like it, could you please at least sort all constants alphabetically ? This way defaults will be close to the properties.

Reply:
👍

Reply:
thanks !